_security/access-control/permissions.md

```diff
@@ -393,3 +393,19 @@ These permissions apply to an index or index pattern. You might want a user to h
 - indices:monitor/shard_stores
 - indices:monitor/stats
 - indices:monitor/upgrade
+
+
+## Rest Permissions
+
+These permissions apply to rest APIs to control access to the endpoints. Granting access to any of these will allow access to change the crucial operational components of Security plugin.
+NOTE: Allowing access to these endpoints can trigger operational changes in the cluster. Proceed with caution.
+
```

Comment:
If these are cluster permissions, let's format them like the others in the Permissions documentation and add them in the section above index permissions, [Cluster permissions](https://opensearch.org/docs/latest/security/access-control/permissions/#cluster-permissions). So, the whole enchilada:

## Security REST API permissions

Allowing access to these endpoints has the potential to trigger operational changes in the cluster. Proceed with caution.
{: .warning }

See [API]({{site.url}}{{site.baseurl}}/security/access-control/api/) for more information about Security REST APIs.

* cluster:restapi:admin/actiongroups
* cluster:restapi:admin/allowlist
* cluster:restapi:admin/internalusers
* cluster:restapi:admin/nodesdn
* cluster:restapi:admin/roles
* cluster:restapi:admin/rolesmapping
* cluster:restapi:admin/ssl/certs/info
* cluster:restapi:admin/ssl/certs/reload
* cluster:restapi:admin/tenants

Reply:
these permissions are not prefixed with `cluster:`, so these should be fine.